js/src/recognizer.ts

```diff
@@ -75,13 +74,6 @@ async function getDetailedLanguage(language: string, files: string[]): Promise<L
     }
 }
 
-async function getFiles(root: string): Promise<string[]> {
-    return new Promise((resolve, reject) => {
-        glob(`${root}/**/*.*`, `!${root}/**`, (err: any, files: string[] | PromiseLike<string[]>) => {
-            if (err) {
-                reject(`Unable to retrieve files from current project ${root}. ${err}`);
-            }
-            return resolve(files);
-        })
-    });
-}
\ No newline at end of file
+function getFiles(root: string): Promise<string[]> {
```

Comment:
Shouldn't this be async?

Reply:
No, if we not call `await` inside function body

Reply:
Right now it is, look at line 22